stixcore/ephemeris/manager.py

```diff
@@ -0,0 +1,111 @@
+from datetime import datetime
+from pathlib import Path
+
+import spiceypy as spice
+from astropy.time.core import Time
+
+SOLAR_ORBITER_NAIF_ID = -144
+
+__all__ = ['SpiceManager']
+
+
+class SpiceManager:
+    """
+    Load the SolarOrbiter kernels as provided in the given meta kernel
+    """
+    def __init__(self, meta_kernel_path):
+        """
+        Load the spice kernel or kernels referenced in the give kernel file
+
+        Parameters
+        ----------
+        meta_kernel_path : `str` or `pathlib.Path`
+            Path to the meta kernel
+
+        """
+        self.mk_path = Path(meta_kernel_path)
+        *_, datestamp, version = self.mk_path.name.split('_')
+        self.kernel_date = datetime.strptime(datestamp, '%Y%m%d')
+        spice.furnsh(str(meta_kernel_path))
```

Comment:
I think as spicepy is a wrapper around the CSPICE library it appears to be global in that kernels loaded in one module / function will be available anywhere spicepy is imported. A actual context manager might be a better approach but not sure.